### ISSUE TITLE
[BUG] RHDA is attempting to put a slash at the beginning of the path to my POM, causing the help:effective-pom mvn command to fail

### ISSUE THREAD

**Issue by natereprogle** [fabric8-analytics/fabric8-analytics-vscode-extension#681]
**Describe the bug**
Every time I focus on my pom.xml in my project, I get the following error:
```
Command failed: mvn -q help:effective-pom -Doutput="\v:\Development Projects\<project-here>\effective-pom.xml" -f "/v:/Development Projects/<project-here>/pom.xml" POM file "/v:/Development Projects/<project-here>/pom.xml" specified the -f/--file command-line argument does not exist
```

Running this command manually also gives the same error, but removing the `/` from in front of `v:/` in the -f argument generates the effective-pom as expected. I have attempted to look through different configuration settings, update Maven to the latest version, etc. with no results.

**To Reproduce**
1. Simply open any pom.xml in my project. I have two
2. Receive error, nothing happens

**Expected behavior**
The effective-pom.xml is generated, and I am able to analyze dependencies as normal

**Screenshots**
![image](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/assets/4064388/3677faa8-1e9f-4375-ad61-6e4d2c3a9e99)
I am able to control click the file it says doesn't exist and navigate to it within VSCode without issue. This is not a symlink or shortcut, it is the actual file itself.

**Logs**
The only logs I see are in the output window of VSCode, which just state the error I am receiving above.
[log.log](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/files/14187846/log.log)

**VSCode(please complete the following information):**
 - OS: Windows 11 23H2 build 22631.3085
 - VSCode version: 1.86.0
 - Dependency Analytics Version v0.9.2

**Additional context**
The -Doutput flag doesn't need to change, strangely enough. It uses `\` instead of `/`, but the file is generated fine. However, even if I change the -f argument to use `\` instead of `/`, it still fails when run manually, with the same result.

**Comment by enielrodriguez** [comment]
I think I have a problem like yours:
ENOENT: no such file or directory, open '/c:/projects/myproject/pom.xml'

**Comment by ionutcbn** [comment]
Same for me:
`mvn -q help:effective-pom -Doutput="\c:\Users\ionut.ciobanu\source\EIDAS\June Development\eidas-node-repository\EIDAS-SP\effective-pom.xml" -f "/c:\Users\ionut.ciobanu\source\EIDAS\June Development\eidas-node-repository\EIDAS-SP\pom.xml"`
does not work issuing file not found, but:

`mvn -q help:effective-pom -Doutput="c:\Users\ionut.ciobanu\source\EIDAS\June Development\eidas-node-repository\EIDAS-SP\effective-pom.xml" -f "c:\Users\ionut.ciobanu\source\EIDAS\June Development\eidas-node-repository\EIDAS-SP\pom.xml"`
works without any issue.

`mvn -q help:effective-pom -Doutput="\c:\Users\ionut.ciobanu\source\EIDAS\June Development\eidas-node-repository\EIDAS-SP\effective-pom.xml" -f "c:\Users\ionut.ciobanu\source\EIDAS\June Development\eidas-node-repository\EIDAS-SP\pom.xml"` also works, so it seems that only the slash after the -f option is the issue.

**Comment by wuzhi2009** [comment]
me too, so it seems that only the slash after the -f option is the issue.

**Comment by GKPavesi** [comment]
same thing happening here

**Comment by natereprogle** [comment]
> same thing happening here, I've downgraded the extension for now and it is working, but the current version is bugged

Which version did you downgrade to? I'd love to know so I can make the switch. I'm working on moving my projects to Gradle right now, but until then I would love to have this plugin working for my Maven stuff.

**Comment by GKPavesi** [comment]
> > same thing happening here, I've downgraded the extension for now and it is working, but the current version is bugged
> 
> Which version did you downgrade to? I'd love to know so I can make the switch. I'm working on moving my projects to Gradle right now, but until then I would love to have this plugin working for my Maven stuff.

I've downgraded to version 0.7.3 which was released about 3 months ago.
![image](https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/assets/97477451/4415163b-0d00-4749-908e-a0084b3e09ca)
I have not tested version 0.9.0 and version 0.9.1, just downgraded directly to 0.7.3 and realized it was working and left if there, but maybe you can also try these versions to see if they work.

**Comment by windwhim** [comment]
I downgraded to 0.9.0, which fixes the problem.

> > > same thing happening here, I've downgraded the extension for now and it is working, but the current version is bugged
> > 
> > 
> > Which version did you downgrade to? I'd love to know so I can make the switch. I'm working on moving my projects to Gradle right now, but until then I would love to have this plugin working for my Maven stuff.
> 
> I've downgraded to version 0.7.3 which was released about 3 months ago. ![image](https://private-user-images.githubusercontent.com/97477451/305250925-4415163b-0d00-4749-908e-a0084b3e09ca.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgzNTYyODAsIm5iZiI6MTcwODM1NTk4MCwicGF0aCI6Ii85NzQ3NzQ1MS8zMDUyNTA5MjUtNDQxNTE2M2ItMGQwMC00NzQ5LTkwOGUtYTAwODRiM2UwOWNhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjE5VDE1MTk0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ0ZjJmYzA2MmNlOTFiNzllODZlMzk2MWYxOTQwNjE0NTY4Yzc1ZTg3OWVhYTM4YmZjZTU2NDQwMDY3ZDgxMjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gr3deDvpcEheTy7SYqZMTwVMd4vQL36JbP9R09vtljo) I have not tested version 0.9.0 and version 0.9.1, just downgraded directly to 0.7.3 and realized it was working and left if there, but maybe you can also try these versions to see if they work.

**Comment by elTorres** [comment]
Same here: I can´t generate effective pom with extension v0.9.2 because the slash at the beginning of the path. 

If I downgrade to v0.9.0 RHDA don´t work with error "request to https://rhda.rhcloud.com/api/v4/analysis failed, reason: connect ETIMEDOUT 18.119.151.206:443"

**Comment by IlonaShishov** [comment]
Thank you @natereprogle and all for reaching out regarding this issue.
A fix for this bug will be available soon.

**Comment by natereprogle** [comment]
> Thank you @natereprogle and all for reaching out regarding this issue.
> 
> A fix for this bug will be available soon. 

Thank you! That's awesome 👏🏼

**Comment by wuzhi2009** [comment]
> Thank you @natereprogle and all for reaching out regarding this issue. A fix for this bug will be available soon.

Vielen Dank für Ihre harte Arbeit!

**Comment by raffian** [comment]
I too have suffered from this problem for a while now on `0.9.3` . At first I thought it was some kind of curse, how something fundamental like maven was broken. Glad to see I'm not alone  🌝

**Comment by elTorres** [comment]
Hi @IlonaShishov 

2 versions (v0.9.3 and v0.9.4) of the RHDA extension have been published after the fix https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/244, but neither of them includes it.

When do you expect to release a version of RHDA that includes it?

Is it possible to download a packaged version of RHDA LSP Server to manually replace it locally?

Thanks in advance

**Comment by dennis-yemelyanov** [comment]
Same issue. I don't want to disable the extension just to get rid of this error, but it would be great to have a fix

**Comment by sa-schiefer** [comment]
Downgraded to 0.9.1 and it seems to be working without the issue.

**Comment by IlonaShishov** [comment]
@elTorres the fix will be included in the next release, version 0.9.5. You can expect it very soon.

We don't offer pre-packaged versions for manual replacement, but once 0.9.5 is out, you'll be able to update your RHDA extension.

Thanks for your patience.

**Comment by rozerinyildiz** [comment]
Downgrading RHDA fixed my problem too, thank you.

**Comment by aphage** [comment]
v1.30.0 appears again.

**Comment by tommai78101** [comment]
@aphage It is likely because 0.9.5 isn't out publicly.

**Comment by shi0rik0** [comment]
I encountered the same problem on v0.9.4. When will the fix be published?

**Comment by e1y4s** [comment]
Same for me, the bug is still present in 0.9.4. It has been present since 0.9.2, so for now, we have to stay on 0.9.1

**Comment by elTorres** [comment]
> @elTorres the fix will be included in the next release, version 0.9.5. You can expect it very soon.
> 
> We don't offer pre-packaged versions for manual replacement, but once 0.9.5 is out, you'll be able to update your RHDA extension.
> 
> Thanks for your patience.

Any news?

**Comment by IlonaShishov** [comment]
We are still working on the fix for your issue and apologize for the delays. We will notify you as soon as it is available.
Thank you for your patience.

**Comment by IlonaShishov** [comment]
Hello,

This issue has been resolved in the latest release, version 0.9.5. We appreciate your help in improving the project. If you have any further issues or suggestions, please feel free to reach out.

Best regards.
